updatecli/updatecli.d/doctl.yml

```
@@ -0,0 +1,75 @@
+---
+name: Bump the `doctl` CLI version
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  doctl:
+    kind: githubrelease
+    name: Get the latest doctl version
+    spec:
+      owner: "digitalocean"
+      repository: "doctl"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      versionfilter:
+        kind: semver
+    transformers:
+      - trimprefix: v
+
+conditions:
+  dockerfileArgDoctlVersion:
+    name: "Does the Dockerfile have an ARG instruction which key is DOCTL_VERSION?"
+    kind: dockerfile
+    disablesourceinput: true
+    spec:
+      file: Dockerfile
+      instruction:
+        keyword: "ARG"
+        matcher: "DOCTL_VERSION"
+  testCstDoctlVersion:
+    name: "Update the value of DOCTL_VERSION in the test harness"
+    kind: yaml
+    disablesourceinput: true
+    spec:
+      file: "cst.yml"
+      key: "metadataTest.labels[9].key"
+      value: io.jenkins-infra.tools.doctl.version
+
+targets:
+  updateCst:
+    name: "Update the value of DOCTL_VERSION in the test harness"
+    kind: yaml
+    spec:
+      file: "cst.yml"
+      key: "metadataTest.labels[9].value"
+    scmid: default
+  updateDockerfile:
+    name: "Update the value of ARG DOCTL_VERSION in the Dockerfile"
+    kind: dockerfile
+    spec:
+      file: Dockerfile
+      instruction:
+        keyword: "ARG"
+        matcher: "DOCTL_VERSION"
+    scmid: default
+
+actions:
+  default:
+    kind: github/pullrequest
+    scmid: default
+    title: Bump the `doctl` CLI version to {{ source "doctl" }}
+    spec:
+      labels:
+        - dependencies
```

```suggestion
        - enhancement
        - dependencies
```
To be coherent with https://github.com/jenkins-infra/docker-hashicorp-tools/pull/355/files#diff-9d606ded193d8141659fe14a3c26959efc7e6fae04ae927e1e1ca01e80fa493eR76-R77

Otherwise remove `enhancement` label  from `trivy` manifest.